### PR TITLE
feat: integrate cobra with the --cache flag/subcommand for merger

### PR DESCRIPTION
**Pull request by zqueamish**
This integrates the --cache flag as a subcommand to merge. It is not possible to specify a different cache folder. It only defaults to the ./cache folder, which happens when fetching.
Resolves [#58](https://github.com/orgs/NovoNordisk-OpenSource/projects/2/views/1?pane=issue&itemId=58376325)

NB! Please merge the merge command branch (#19), before this, as it depends on it.

### CODE REVIEW

File: src/cmd/merge.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import (
+	"fmt"
+
+	"github.com/NovoNordisk-OpenSource/decentralized-tech-radar/Merger"
+	"github.com/spf13/cobra"
+)
+
+// mergeCmd represents the merge command
+var mergeCmd = &cobra.Command{
+	Use:   "merge \"<FilePath1> <FilePath2> [FilePath3, ...]\"",
```
Comment (Haruyzal) [inline src/cmd/merge.go:15]:
```suggestion
	Use:   "merge <FilePath1> <FilePath2> [FilePath3, ...]",
```

**Review by Haruyzal — COMMENTED** [review]
Other than removing the quotation-marks in the src/cmd/merge.go, it should be gucci methinks

**Review by JonasSkjodt — APPROVED** [review]
looks good other than the small error @Haruyzal pointed out.

**Review by Alex-itu — APPROVED** [review]
Looks good. Approve if the comment gets done.

**Pull request merged**